package.json

```diff
@@ -31,12 +34,15 @@
     "style-loader": "^2.0.0",
     "stylelint": "^13.8.0",
     "stylelint-config-standard": "^19.0.0",
+    "ts-loader": "^8.0.17",
+    "ts-node": "^9.1.1",
+    "typescript": "^4.1.5",
     "webpack": "^5.11.0",
     "webpack-cli": "^3.3.12",
     "webpack-dev-server": "^3.11.0"
   },
   "engines": {
-    "node": ">=13.2.0"
+    "node": ">=14.15.5"
```

Comment:
Is there something specific in this version? Or do you just need `14.x.0` for some x? Mostly asking, because I'm at `14.15.1` and I don't see any reason to make me/everyone upgrade if not needed.

Reply:
Not specific reason, just current LTS version. I think I should downgrade it to 14.15.0.